src/main/java/com/xiao/authserver/entity/User.java

```diff
@@ -10,7 +10,6 @@
 public class User {
 
     @Id
-    @GeneratedValue(strategy= GenerationType.IDENTITY)
```

Comment:
should add xiaoUserId, instead of use primary key

Reply:
Created new field refUserId